api/docs/release.dox

```diff
@@ -143,6 +143,9 @@ changes:
    enum now completely overlaps the DR_REG_ enum.
    This is a binary compatibility change for the OPSZ_ enum.
  - Adds a new encoding hint field to #instr_t.
+ - The dr_mcontext_t struct and all structs that contain it have changed extending the
+   size of its simd structure and appending a new field opmask, and are not backward
+   compatible with releases prior to \DR_VERSION.
```

Comment:
But this is the opposite of what we would normally do: we would use the struct's size field to keep backward compatibility.  I thought that was the plan.  We put this size field in place precisely to *avoid* any compatibilty break.

Reply:
Comment was stale. Removed.